substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ReflectionConfigurationParser.java

```diff
@@ -44,9 +44,15 @@
     private static final String CONSTRUCTOR_NAME = "<init>";
 
     private final ReflectionConfigurationParserDelegate<T> delegate;
+    private final boolean allowIncompleteClasspath;
 
     public ReflectionConfigurationParser(ReflectionConfigurationParserDelegate<T> delegate) {
+        this(delegate, false);
```

Comment:
@peter-hofer should the `ReflectionConfigurationParser` instance in `com.oracle.svm.configure.config.ConfigurationSet.loadTypeConfig()` be configured with `allowIncompleteClasspath` too? The reflection config there is the result of profiling, could it stil throw on missing classes?

Reply:
That code only builds an in-memory reflection configuration without doing any lookups on the class path, so it should not throw.